astropy/samp/hub_script.py

```diff
@@ -106,16 +107,18 @@ def hub_script(timeout=0):
         "specify the output files where redirect the logging messages.",
     )
 
+    log_levels = sorted(d:=logging.getLevelNamesMapping(), key=d.get, reverse=True) # reverse ordered by level
```

Comment:
putting the comment on a separate line will help with formatting (required)
```suggestion
    # reverse ordered by level
    log_levels = sorted(d := logging.getLevelNamesMapping(), key=d.get, reverse=True)
```